app/models/resque_instance.rb

```diff
@@ -135,6 +135,30 @@ def kill_worker(worker_id)
     raise
   end
 
+  # Get the schedule, if present, as an array of ScheduleElement instances
+  #
+  # This relies on the internals of resque-scheduler's implementation.  Because it
+  # must depend on the single, global Resque instances, we can't use that, so we have to reach
+  # into the underlying redis.  Ugh.
+  def schedule
+    resque_schedule = @resque_data_store.hgetall("schedules")
```

Comment:
This is the only method that depends on the internals of resque-scheduler.  Basically the format it uses and the redis key where it keeps the schedule.